index.d.ts

```diff
@@ -34,4 +34,5 @@ export function snakeCase (str: string): string
 export function sum (values: Array<string>): number
 export function sumBy (values: Array, iteratee: Function | string): number
 export function uniqBy (array: Array, iteratee: Function | string): Array
-export function without (array: Array, ...values: Array): Array
+export const validation : { NUMBER: RegExp; EMAIL: RegExp ; PATH: RegExp; NAME: RegExp; NAME_WITH_DIGITS: RegExp; INPUT: RegExp; ADDRESS: RegExp; PHONE_CODE: RegExp; IMAGE: RegExp; FILE: RegExp; FILENAME: RegExp; ADMIN_PASSWORD: RegExp }
+export function without (array: Array, ...values: Array): Array
```

Comment:
hmm I see newline missing at end of file, please restore it

Reply:
The line is present, but somehow Github review don't show the last empty line.